04.ls/ls.rb

```diff
@@ -33,4 +33,119 @@ def split_files(files)
   divided_files
 end
 
+HALF_YEAR = 15_552_000
+
+FILE_TYPE = {
```

Comment:
```suggestion
FILE_TYPES = {
```

ハッシュも配列と同じく複数形にすることが多いですね。（単数形になってるケースもときどきありますが）

Reply:
同じく複数形に変更しました。